PowerShell/ScubaGear/Modules/Orchestrator.psm1

```diff
@@ -1046,14 +1049,22 @@ function Merge-JsonOutput {
 
                 # The date is listed under the metadata, no need to include it in the summary as well
                 $IndividualResults.ReportSummary.PSObject.Properties.Remove('Date')
-
                 $Summary | Add-Member -NotePropertyName $BaselineName `
                     -NotePropertyValue $IndividualResults.ReportSummary
             }
+            foreach ($Product in $Results.PSObject.Properties) {
+                foreach ($Group in $Product.Value) {
+                    foreach ($Control in $Group.Controls) {
+                        $Control.Requirement = Format-PlainText -RawString $Control.Requirement
+                        $Control.Details = Format-PlainText -RawString $Control.Details
+                    }
+                }
+            }
 
             # Convert the output a json string
             $MetaData = ConvertTo-Json $MetaData -Depth 3
             $Results = ConvertTo-Json $Results -Depth 5
+           # Loop through each property
```

Comment:
A very very minor thing...I don't understand this comment. There's no looping here. Maybe a copy-paste mistake? Or maybe I'm missing something obvious.

Reply:
I took out the comment.  I think I had started doing something and then changed my approach so it's no longer relevant